src/pb_stub.cc

```diff
@@ -1613,6 +1602,97 @@ PYBIND11_EMBEDDED_MODULE(c_python_backend_utils, module)
       module, "TritonModelException");
 }
 
+
+void
+ModelContext::Init(
+    const std::string& model_path, const std::string& platform,
+    const std::string& triton_install_path, const std::string& model_version)
+{
+  bool python_model_found = false;
+  std::string platform_model_path;
+
+  if (platform != "NONE") {
+    platform_model_path =
+        triton_install_path + "/platform_handlers/" + platform + "/model.py";
+    // Check if model file exists in the path.
+    struct stat buffer;
+    if (stat(platform_model_path.c_str(), &buffer) == 0) {
+      // Use the Platform model for serving the model.
+      python_model_found = true;
+      type_ = ModelType::PLATFORM;
+      python_model_path_ = platform_model_path;
+      // Trimming the model name from the model path, the platform model
+      // will populate the expected default model file name into model_path_.
+      model_path_ = model_path.substr(0, model_path.find_last_of("\\/"));
+    } else {
+      LOG_WARN << "Unable to find model(handler) \'" << platform_model_path
+               << "\' for platform field \'" << platform << "\'";
+    }
+  }
+
+  if (!python_model_found) {
+    python_model_path_ = model_path;
```

Comment:
potential: `python_module_path` to distinguish from `model`

Reply:
I would prefer python_model_path for now... It makes more sense as we have always referred model.py as python model in Triton.